File: KarateKlub/KarateKlub.Api/Models/RezultatiTakmicenjaTrenerPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class RezultatiTakmicenjaTrenerPregledVM
    {
        public class Row {
            public int id;
            public int takmicarId;
            public string takmicar;
            public string starosnaDob;
            public string kategorija;
            public string disciplina;
            public string osvojenoMjesto;
            public int brojTakmicaraUKategoriji;
            public int brojPobjeda;
            public int brojPoraza;

        }
        public List<Row> rows;
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_trener/model/RezultatiTakmicenjaTrenerPregledVM.java
package com.example.x.karateklub.area_trener.model;

import java.io.Serializable;
import java.util.List;

public class RezultatiTakmicenjaTrenerPregledVM implements Serializable {
    public  static class Row implements  Serializable{
        public int id;
        public int takmicarId;
        public String takmicar;
        public String starosnaDob;
        public String kategorija;
        public String disciplina;
        public String osvojenoMjesto;
        public int brojTakmicaraUKategoriji;
        public int brojPobjeda;
        public int brojPoraza;


    }
public List<Row> rows;

}
